src/Dfc.CourseDirectory.WebV2/Features/NewApprenticeshipProvider/NewApprenticeshipProviderController.cs

```diff
@@ -115,5 +115,14 @@ public async Task<IActionResult> ProviderDetailConfirmation(ProviderInfo provide
                             .WithMptxInstanceId(Flow)
                     }));
         }
+
+
+        [HttpPost("hide-passed-notification")]
+        public async Task<IActionResult> HidePassedNotication([LocalUrl] string returnUrl, ProviderInfo providerInfo, HidePassedNotification.Command command)
```

Comment:
Don't need `ProviderInfo` action parameter here; can use `ProviderContext` property

Reply:
didn't notice the ProviderContext. Done :)